includes/wp-admin/class-settings.php

```diff
@@ -465,42 +557,267 @@ public static function add_settings_help_tab() {
+					'</dl>' . "\n" .
+
+					'<h2>' . \esc_html__( 'Link and Permalink Tags', 'activitypub' ) . '</h2>' . "\n" .
+					'<dl>' . "\n" .
+					'<dt><code>[ap_permalink type="url"]</code></dt>' . "\n" .
+					'<dd>' . \wp_kses( \__( 'The post&#8217;s permalink. <code>type</code> can be either: <code>url</code> or <code>html</code> (an &lt;a /&gt; tag). <code>type</code> attribute is optional.', 'activitypub' ), $code_html ) . '</dd>' . "\n" .
+					'<dt><code>[ap_shortlink type="url"]</code></dt>' . "\n" .
+					'<dd>' . \wp_kses( \__( 'The post&#8217;s shortlink. <code>type</code> can be either <code>url</code> or <code>html</code> (an &lt;a /&gt; tag). I can recommend <a href="https://wordpress.org/plugins/hum/" target="_blank">Hum</a>, to prettify the Shortlinks. <code>type</code> attribute is optional.', 'activitypub' ), $code_html ) . '</dd>' . "\n" .
+					'</dl>' . "\n" .
+
+					'<h2>' . \esc_html__( 'Metadata Tags', 'activitypub' ) . '</h2>' . "\n" .
+					'<dl>' . "\n" .
+					'<dt><code>[ap_hashtags]</code></dt>' . "\n" .
+					'<dd>' . \esc_html__( 'The post&#8217;s tags as hashtags.', 'activitypub' ) . '</dd>' . "\n" .
+					'<dt><code>[ap_hashcats]</code></dt>' . "\n" .
+					'<dd>' . \esc_html__( 'The post&#8217;s categories as hashtags.', 'activitypub' ) . '</dd>' . "\n" .
+					'<dt><code>[ap_author]</code></dt>' . "\n" .
+					'<dd>' . \esc_html__( 'The author&#8217;s name.', 'activitypub' ) . '</dd>' . "\n" .
+					'<dt><code>[ap_authorurl]</code></dt>' . "\n" .
+					'<dd>' . \esc_html__( 'The URL to the author&#8217;s profile page.', 'activitypub' ) . '</dd>' . "\n" .
+					'<dt><code>[ap_date]</code></dt>' . "\n" .
+					'<dd>' . \esc_html__( 'The post&#8217;s date.', 'activitypub' ) . '</dd>' . "\n" .
+					'<dt><code>[ap_time]</code></dt>' . "\n" .
+					'<dd>' . \esc_html__( 'The post&#8217;s time.', 'activitypub' ) . '</dd>' . "\n" .
+					'<dt><code>[ap_datetime]</code></dt>' . "\n" .
+					'<dd>' . \esc_html__( 'The post&#8217;s date/time formated as "date @ time".', 'activitypub' ) . '</dd>' . "\n" .
+					'</dl>' . "\n" .
+
+					'<h2>' . \esc_html__( 'Site Information Tags', 'activitypub' ) . '</h2>' . "\n" .
+					'<dl>' . "\n" .
+					'<dt><code>[ap_blogurl]</code></dt>' . "\n" .
+					'<dd>' . \esc_html__( 'The URL to the site.', 'activitypub' ) . '</dd>' . "\n" .
+					'<dt><code>[ap_blogname]</code></dt>' . "\n" .
+					'<dd>' . \esc_html__( 'The name of the site.', 'activitypub' ) . '</dd>' . "\n" .
+					'<dt><code>[ap_blogdesc]</code></dt>' . "\n" .
+					'<dd>' . \esc_html__( 'The description of the site.', 'activitypub' ) . '</dd>' . "\n" .
+					'</dl>' . "\n" .
+
+					'<h2>' . \esc_html__( 'Template Examples', 'activitypub' ) . '</h2>' . "\n" .
+					'<p>' . \esc_html__( 'Basic post template:', 'activitypub' ) . '</p>' . "\n" .
+					'<pre>[ap_title]
```

Comment:
we should either design that as a box or use a textarea for it.

Reply:
Do we even want to add that in the first place?